src/plugins/typechecker/README.md

```diff
@@ -0,0 +1,40 @@
+- infos = Information about the typechecker plugin is in keys below
+- infos/author = e1528532 <e1528532@libelektra.org>
+- infos/licence = BSD
+- infos/needs = 
+- infos/provides = typechecker
+- infos/recommends =
+- infos/placements = postgetstorage presetstorage
+- infos/status = maintained experimental global
+- infos/metadata =
+- infos/description = a plugin which typechecks configuration specifications
+
+## Introduction
+
+A plugin which typechecks specifications before setting keys and after getting keys from
+a mounted specification. 
+
+## Usage
+
+To see its functionality there will be a test specification installed into 
```

Comment:
Usage should be described here and not immediately referred to some other document.

Reply:
I didn't want to spend too much effort on the usage yet as a lot of things are still likely to change, right now its more intended as a proof of concept for you and others who may are interested in it.